tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -184,119 +208,52 @@ func (rb *RBACAdditionalTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
 	assert.Equal(rb.T(), 0, len(userGetProject.Data))
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateListGlobalSettings() {
-	adminListSettings, err := listGlobalSettings(rb.steveAdminClient)
+func (rb *RBACAdditionalTestSuite) TestRBACP1RestrictedAdmin() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	_, restrictedAdminClient, err := setupUser(rb.client, restrictedAdmin)
+	require.NoError(rb.T(), err)
+	rb.T().Logf("Validating restricted admin can create an RKE1 cluster")
+	userConfig := new(provisioninginput.Config)
+	config.LoadConfig(provisioninginput.ConfigurationFileKey, userConfig)
+	nodeProviders := userConfig.NodeProviders[0]
+	nodeAndRoles := []provisioninginput.NodePools{
+		provisioninginput.AllRolesNodePool,
+	}
+	externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviders)
+	clusterConfig := clusters.ConvertConfigToClusterConfig(userConfig)
+	clusterConfig.NodePools = nodeAndRoles
+	kubernetesVersion, err := kubernetesversions.Default(restrictedAdminClient, clusters.RKE1ClusterType.String(), []string{})
+	require.NoError(rb.T(), err)
+
+	clusterConfig.KubernetesVersion = kubernetesVersion[0]
+	clusterConfig.CNI = userConfig.CNIs[0]
+	clusterObject, _, err := provisioning.CreateProvisioningRKE1CustomCluster(restrictedAdminClient, &externalNodeProvider, clusterConfig)
+	require.NoError(rb.T(), err)
+	provisioning.VerifyRKE1Cluster(rb.T(), restrictedAdminClient, clusterConfig, clusterObject)
+
+	rb.T().Log("Validating restricted Admin can list global settings")
+	//Steve client is required to list global settings.
+	steveRestrictedAdminclient := restrictedAdminClient.Steve
+	steveAdminClient := rb.client.Steve
+
+	adminListSettings, err := listGlobalSettings(steveAdminClient)
 	require.NoError(rb.T(), err)
-	resAdminListSettings, err := listGlobalSettings(rb.steveStdUserclient)
+	resAdminListSettings, err := listGlobalSettings(steveRestrictedAdminclient)
 	require.NoError(rb.T(), err)
 
 	assert.Equal(rb.T(), len(adminListSettings), len(resAdminListSettings))
 	assert.Equal(rb.T(), adminListSettings, resAdminListSettings)
-}
 
-func (rb *RBACAdditionalTestSuite) ValidateEditGlobalSettings() {
-	kubeConfigTokenSetting, err := rb.steveStdUserclient.SteveType("management.cattle.io.setting").ByID(kubeConfigTokenSettingID)
+	rb.T().Logf("Validating restrictedAdmin can edit global settings")
```

Comment:
Should this be "rb.T().Log("Validating restricted Admin CANNOT edit global settings")?